<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayer</class>
 <widget class="QWidget" name="VideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="8,1,1">
   <item>
    <widget class="QWidget" name="PlayWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="SpeedControl">
     <property name="title">
      <string>速度控制</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="FastButton">
        <property name="text">
         <string>正常</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="SlowButton">
        <property name="text">
         <string>慢速</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="VerySlowButton">
        <property name="text">
         <string>超慢速</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="MirrorControl">
     <property name="title">
      <string>示范类型</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="NormalButton">
        <property name="text">
         <string>正面示范</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="MirrorButton">
        <property name="text">
         <string>镜面示范</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
